### ISSUE TITLE
python library as separate repository

### ISSUE THREAD

**Issue by juhi24** [fmidev/opendata-resources#1]
In my opinion, it would make sense to have the open data python library, currently deep in the examples folder hierarcy, as a separate git repository. That way, it would be easier to find and deploy, among other things.

**Comment by tervo** [comment]
You are definitely right and we will do this at some point. Anyway, some refactoring and documentation is clearly required before one can declare this as an library in good conscience.

**Comment by juhi24** [comment]
I agree that it makes sense to do any major refactoring before publishing this as a separate python module, especially if you already have some sort of plan in mind on what to do. On the other hand, please keep in mind that potentially there are people willing to help with the development, and the earlier you publish this in an accessible way (=separate repo), the sooner people will find it and start contributing.

I made #2 to demonstrate this ;)

**Comment by tervo** [comment]
Thank you very much for the contribution! You are definitely right.

I'll write the most important issues (and hopefully solve at least part of them) and move the code to separate module as soon as possible.
